<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="2" ignored="0" total="3" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-06-29T11:07:07 IST" name="Suite" finished-at="2024-06-29T11:08:27 IST" duration-ms="79317">
    <groups>
    </groups>
    <test started-at="2024-06-29T11:07:07 IST" name="Test" finished-at="2024-06-29T11:08:27 IST" duration-ms="79317">
      <class name="Campaigns.CreateCampaignTestWithTakesScreenshot">
        <test-method is-config="true" signature="bs()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:07:07 IST" name="bs" finished-at="2024-06-29T11:07:07 IST" duration-ms="13" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bs -->
        <test-method is-config="true" signature="bt()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:07:08 IST" name="bt" finished-at="2024-06-29T11:07:08 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bt -->
        <test-method is-config="true" signature="bc()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:07:08 IST" name="bc" finished-at="2024-06-29T11:07:30 IST" duration-ms="21978" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bc -->
        <test-method is-config="true" signature="bm()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:07:30 IST" name="bm" finished-at="2024-06-29T11:07:51 IST" duration-ms="20871" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bm -->
        <test-method retried="true" signature="CreateCampaignTest()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:07:51 IST" name="CreateCampaignTest" finished-at="2024-06-29T11:08:02 IST" duration-ms="11350" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:819)
at org.testng.Assert.assertEquals(Assert.java:829)
at Campaigns.CreateCampaignTestWithTakesScreenshot.CreateCampaignTest(CreateCampaignTestWithTakesScreenshot.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateCampaignTest -->
        <test-method is-config="true" signature="am()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:08:02 IST" name="am" finished-at="2024-06-29T11:08:04 IST" duration-ms="2320" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- am -->
        <test-method is-config="true" signature="bm()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:08:04 IST" name="bm" finished-at="2024-06-29T11:08:08 IST" duration-ms="3445" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bm -->
        <test-method retried="true" signature="CreateCampaignTest()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:08:08 IST" name="CreateCampaignTest" finished-at="2024-06-29T11:08:15 IST" duration-ms="6879" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:819)
at org.testng.Assert.assertEquals(Assert.java:829)
at Campaigns.CreateCampaignTestWithTakesScreenshot.CreateCampaignTest(CreateCampaignTestWithTakesScreenshot.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateCampaignTest -->
        <test-method is-config="true" signature="am()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:08:15 IST" name="am" finished-at="2024-06-29T11:08:16 IST" duration-ms="1386" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- am -->
        <test-method is-config="true" signature="bm()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:08:16 IST" name="bm" finished-at="2024-06-29T11:08:19 IST" duration-ms="3214" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bm -->
        <test-method signature="CreateCampaignTest()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:08:19 IST" name="CreateCampaignTest" finished-at="2024-06-29T11:08:25 IST" duration-ms="5493" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:819)
at org.testng.Assert.assertEquals(Assert.java:829)
at Campaigns.CreateCampaignTestWithTakesScreenshot.CreateCampaignTest(CreateCampaignTestWithTakesScreenshot.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateCampaignTest -->
        <test-method is-config="true" signature="am()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:08:25 IST" name="am" finished-at="2024-06-29T11:08:27 IST" duration-ms="1600" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- am -->
        <test-method is-config="true" signature="ac()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:08:27 IST" name="ac" finished-at="2024-06-29T11:08:27 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ac -->
        <test-method is-config="true" signature="at()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:08:27 IST" name="at" finished-at="2024-06-29T11:08:27 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- at -->
        <test-method is-config="true" signature="as()[pri:0, instance:Campaigns.CreateCampaignTestWithTakesScreenshot@5677323c]" started-at="2024-06-29T11:08:28 IST" name="as" finished-at="2024-06-29T11:08:28 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- as -->
      </class> <!-- Campaigns.CreateCampaignTestWithTakesScreenshot -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
